tests/test_modeling_ctrl.py

```diff
@@ -41,7 +41,7 @@ class CTRLModelTester(object):
         def __init__(
             self,
             parent,
-            batch_size=13,
+            batch_size=14,
```

Comment:
We need an even number of `batch_size` ? Or is it because it's an unlucky number? :D

Reply:
Yes for some (but not all for some reason) of the models, the batch size seems to need to be a multiple of the number of DataParallel replicas.

I didn’t investigate too much as to why.